### DISCUSSION TITLE
Sort and file/directory organization

### DISCUSSION THREAD

**Discussion by llagerlof** [antonmedv/llama#31] [Ideas]
First, I think your program is very nice, and it's almost exactly what I was searching for: a fast and minimalistic way to navigate my directories. Currently I am using a bash script I wrote using `fzf`, but a binary solution would be nice.

So, I have some UX suggestions. I will give them and provide the reason for each one:

**Sort, by default, by "Modify/Create" date/time. Last created first.**

It's very easy to be lost when the directory list is big. If the last created directories appear first in the list the user could access his most recent directories without the need to remember the exact name or type anything (the fuzzy finder is still useful for older directories in that scenario). The fact that your program focus on first item is good because with just one ENTER the user can access his last created directory.

**Show directories first, files after**

If the main goal of this application is to allow quick navigation between directories, it's common sense to show them first.

**Show the full path instead the relative one in the title**

So it's easier to copy the full path if the user needs it.

Of course, this are the minimum sensible defaults for the application's goal, but each user has different needs, so I guess all this could be in a config file.

Thanks!

**Comment by antonmedv** [comment]
Thanks for suggestions. I was thinking about ordering, for beginners decided to just use alphabetical order.

Let's add something like LLAMA_SORT_BY= env.
